src/terms.jl

```diff
@@ -567,13 +567,15 @@ vectorize(x) = [x]
 """
     coefnames(term::AbstractTerm)
 
-Return the name(s) of column(s) generated by a term.  Return value is either a
-`String` or an iterable of `String`s.
+Return the name(s) of column(s) generated by a term.
+
+Return value is either a `String`, an iterable of `String`s or the empty vector 
+if there is no associated column (e.g. `coefnames(InterceptTerm{false}())`).
```

Comment:
```suggestion
Return value is either a `String`, an iterable of `String`s, the empty vector 
if there is no associated column (e.g. `coefnames(InterceptTerm{false}())`), or
(in the case of a `FormulaTerm`) a two tuple of any of the above (corresponding 
to `coefnames` applied to the left- and right-hand sides of the formula).
```

Reply:
What if I copied and adapted the docstring for `termnames(t::FormulaTerm)` instead? My goal was to make the two functions consistent, and the behavior for formulas seems different enough to deserve a separate docstring.